cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -247,8 +257,21 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate ORT re
             ortResultInput = ortResultInput.replaceConfig(config)
         }
 
-        packageCurationsFile?.let {
-            val curations = it.readValueOrDefault(emptyList<PackageCuration>())
+        val curationFiles = mutableListOf<File>()
+        packageCurationsFile?.takeIf { it.isFile }?.let { curationFiles += it }
+        packageCurationsDir?.let { curationFiles += FileFormat.findFilesWithKnownExtensions(it) }
+
+        val curations = curationFiles.mapNotNull { curationFile ->
+            runCatching {
```

Comment:
The lines from 265 - 271 are redundant with the ones existing in `FilePackageCurationProvider`, except for one inconsistency. Please remove the duplication.

e.g. would it make sense to refactor it to end up with something like below?
```
val curationsFiles = FilePackageCurationProvider.readFile(packageCurationsFile) 
   + FilePackageCurationProvider.readFile(packageCurationsFile).readDir(packageCurationsDir)
```

Reply:
Maybe even such that there is no more redundancy regarding the ordering, like e.g.

`FilePackageCurationProvider.for(file=packageCurationsDir, dir=packageCurationsFile)`

Reply:
I refactored it.
The calling code now looks like this:
`val curations = FilePackageCurationProvider.from(packageCurationsFile, packageCurationsDir).packageCurations`

Not completely happy with making the parameters `from` nullable, but all solutions I could think of would make the calling code pretty ugly.